src/install/install.zig

```diff
@@ -713,6 +707,15 @@ const PackageInstall = struct {
                     this.package_install.cache_dir_subpath = folder_buf[0 .. "../".len + folder.len :0];
                     this.package_install.cache_dir = std.fs.cwd();
                 },
+                .workspace => {
+                    var folder_buf = &cache_dir_subpath_buf;
+                    const folder = resolution.value.workspace.slice(ctx.string_buf);
+                    std.mem.copy(u8, folder_buf, "../" ++ std.fs.path.sep_str);
+                    std.mem.copy(u8, folder_buf["../".len..], folder);
+                    folder_buf["../".len + folder.len] = 0;
+                    this.package_install.cache_dir_subpath = folder_buf[0 .. "../".len + folder.len :0];
```

Comment:
we can't assume a specific number of levels above a directory. the workspace can be anywhere in theory

Reply:
oh this is just package.json, ignore